pom.xml

```diff
@@ -205,7 +188,7 @@
             <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
         </repository>
         <snapshotRepository>
-            <id>sonatype-nexus-snapshots</id>
+            <id>sonatype-nexus-staging</id>
```

Comment:
Just asking.  I noticed this name was changed from snapshots to staging but the url remains to snapshots.  Is this correct?

Reply:
It is. The Sonatype release manager manages which release goes where based on the `SNAPSHOT` suffix. More info [here](https://central.sonatype.org/publish/publish-maven/#distribution-management-and-authentication)